### ISSUE TITLE
Lesson 1.03: Cloud9 - Python summary

### ISSUE THREAD

**Issue by normanbauer** [TEALS-IntroCS/2nd-semester-introduction-to-computer-science-principles#44]
Is there a summary of how Cloud9 works with Python?  The colors of the commands, variable completion, etc.  If there is I believe a summary/cheat sheet or a lesson for these would of great benefit as I know my students will not have had any experience with an IDE. I have already created an issue about the interactive/BASH part of the Cloud9 Python IDE.

**Comment by normanbauer** [comment]
Also the indentations have to be done manually and a return to terminate the indentation.

**Comment by TEALS-IntroCS** [comment]
When we introduce Script Mode, the lesson plan should highlight some of the IDE features:
-what do the different color-coding mean?
-how do you indent and unindent?
-what sort of things will auto-complete?

**Comment by goldblatt** [comment]
I added a cheat sheet to this lesson with a few basic comamnds to open/close files. I think we could include a few other things in it as well. just not sure if that makes the lesson too long

**Comment by normanbauer** [comment]
I really think this is very important.  I know my students have never seen an IDE before and getting them well grounded and familiar with it will really help those who go on in Computer Science. Those who don't, I still think it will be a valuable tool for them in their careers.

**Comment by TEALS-IntroCS** [comment]
I agree this is important. On second thought, this might make the most sense as a quick 3-minute video showing the features of the IDE. We'll work on that. It isn't the top priority right now.

**Comment by TEALS-IntroCS** [comment]
We should be sure to mention the ability to change themes for different color coding.

**Comment by TEALS-IntroCS** [comment]
Making a movie this evening, will upload to YouTube and then create link in this lesson.

**Comment by normanbauer** [comment]
Great, I think it will be very valuable

**Comment by smeredith** [comment]
Did a movie ever materialize?
